CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 2.13.0 (2018-10-23)
+- Updated the [skyux-builder-plugin-stache](https://github.com/blackbaud/skyux-builder-plugin-stache/blob/master/CHANGELOG.md#151-2018-08-02) to `1.7.0`.
```

Comment:
I'd either delete "the" or add a noun after "skyux-builder-plugin-stache." Is this a dependency? If so, maybe "dependency" would work.